main.py

```diff
@@ -98,6 +100,18 @@ def get_encoder_schema(feature: str, type: str):
     def get_decoder_schema(feature: str, type: str):
         return get_decoder_cls(feature, type)
 
+    @env.macro
+    def get_split_schema(type: str, col_name: str = None):
+        cls = get_split_cls(type)
+        # Random split doesn't have column as an attribute
+        if type != "random" and col_name:
```

Comment:
You probably don't want to set this on the class, rather you want to set it on the default object as it's being converted into YAML. That's what the `updates` param to [schema_class_to_yaml](https://github.com/ludwig-ai/ludwig-docs/pull/263/files#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R122) does, so in your case you could do something like:

```
schema_class_to_yaml(..., updates={"name": "my_column_name"})
```

Reply:
Nice, fixed!